src/main/resources/MAT.properties

```diff
@@ -17,8 +17,8 @@ mat.password.expiry.email.subject="Your MAT Account "
 mat.from.emailAddress=sb-mat-noreply-help@hcqis.org
 mat.support.emailAddress=sb-mat-help@semanticbits.com
 
-mat.privacypolicy.url=http://www.cms.gov/About-CMS/Agency-Information/Aboutwebsite/Privacy-Policy.html
-mat.termsofuse.url=https://www.emeasuretool.cms.gov/web/guest/terms-of-use
+mat.privacypolicy.url=https://www.cms.gov/About-CMS/Agency-Information/Aboutwebsite/Policiesforaccessibility
+mat.termsofuse.url=https://harp.cms.gov/login/terms-of-use
 mat.accessibilitypolicy.url=https://www.emeasuretool.cms.gov/web/guest/accessibility-statement

```

Comment:
AccessibilityPolicy isn't working? I know its not a part of change, but I wonder if we have to fix it as well

Reply:
Nice catch! I had updated all three, or so I thought. Turns out I pasted over the updated privacy policy with the updated accessibility policy link.